diff --git a/make/autoconf/build-aux/config.guess b/make/autoconf/build-aux/config.guess
index 52e598d..f4c6d17 100644
--- a/make/autoconf/build-aux/config.guess
+++ b/make/autoconf/build-aux/config.guess
@@ -67,15 +67,6 @@ if test $? = 0; then
   esac
 fi
 
-# Test and fix wsl
-echo $OUT | grep '\(unknown\|pc\)-linux-gnu' > /dev/null 2> /dev/null
-if test $? = 0; then
-  uname -r | grep -i microsoft > /dev/null 2> /dev/null
-  if test $? = 0; then
-    OUT=`echo $OUT | sed -e 's/\(unknown\|pc\)-linux-gnu/pc-wsl/'`
-  fi
-fi
-
 # Test and fix architecture string on AIX
 # On AIX 'config.guess' returns 'powerpc' as architecture but 'powerpc' is
 # implicitly handled as 32-bit architecture in 'platform.m4' so we check
