diff --git a/src/hotspot/os/posix/os_posix.cpp b/src/hotspot/os/posix/os_posix.cpp
index 7f95560..6e8094c 100644
--- a/src/hotspot/os/posix/os_posix.cpp
+++ b/src/hotspot/os/posix/os_posix.cpp
@@ -2031,7 +2031,7 @@ void os::abort(bool dump_core, void* siginfo, const void* context) {
     LINUX_ONLY(if (DumpPrivateMappingsInCore) ClassLoader::close_jrt_image();)
     ::abort(); // dump core
   }
-  os::_exit(1);
+  ::raise(SIGKILL);
 }
 
 // Die immediately, no exit hook, no abort hook, no cleanup.
